src/sys/inotify.rs

```diff
@@ -137,6 +117,43 @@ bitflags! {
     }
 }
 
+#[derive(Debug)]
+pub struct InotifyEvent {
+    pub wd: i32,
```

Comment:
What do these fields mean?  If they're going to be public, then they should be documented, especially the enigmatic `wd` and `cookie`.